shared/src/main/scala/Elements.scala

```diff
@@ -623,6 +623,24 @@ object Elements:
         MiscHelpers.reduce(a, b)
     }
 
+    val sort: Monad = addElem(
+      Monad,
+      "S",
+      "Sort ascending",
+      List(
+        "sort",
+        "sortasc",
+        "sort-asc",
+      ),
+      "a: any -> convert to list and sort ascending",
+    ) {
+      // should do something else for num overload later
+      case (a: VAny) =>
+        VList(
+          ListHelpers.makeIterable(a).sorted(MiscHelpers.compareExact(_, _))*
+        )
```

Comment:
We should decide on a style for this sort of thing. Should `VList.from(x)` be preferred or `VList(x*)`? `VList.from` is more verbose, but I like it more because I'm not sure if `*` creates a new `ArraySeq` or something. If it does, it might not only be bad for performance but also cause an infinite lazy list to be evaluated. Gonna have to look this up.